src/v0/destinations/hs/HSTransform-v1.js

```diff
@@ -232,7 +234,7 @@ const batchIdentifyForrETL = (
         metadata.push(ev.metadata);
       });
     } else {
-      throw new CustomError("[HS]:: rETL -  Unknow hubspot operation", 400);
+      throw new TransformationError("rETL -  Unknow hubspot operation", 400);
```

Comment:
This should not be a transformation error. Check the logic once again.

Reply:
We are setting that field by default. What should be the correct error category then?